ast/ast_test.go

```diff
@@ -21,12 +23,23 @@ func TestAstModelParses(t *testing.T) {
 func TestAstOverrides(t *testing.T) {
 	model := Parse("test/model.go")
 
-	model.parseOverrides()
-
```

Comment:
model.parseOverrides(), calls parseOverrides on each field. Since we're explicitly calling field.parseOverrides below, this call wasn't necessary.